train.py

```diff
@@ -58,6 +64,9 @@ def get_args():
     parser.add_argument('--saved_path', type=str, default='logs/')
     parser.add_argument('--debug', type=bool, default=False, help='whether visualize the predicted boxes of trainging, '
                                                                   'the output images will be in test/')
+    parser.add_argument('--apex', help='Train using AMP with Apex', action='store_true')
+    parser.add_argument('--opt_level', type=str, default='O1', help='Opt level for Apex AMP')
+    parser.add_argument('--clip_grad', type=float, default=10.0, help='Clip gradient norm used by clip_grad_norm_')                                                                  
```

Comment:
These three parameters are used when training with apex. Not specifying any of them will allow training as usual